devito/passes/clusters/blocking.py

```diff
@@ -65,6 +72,9 @@ def callback(self, clusters, prefix):
 
         d = prefix[-1].dim
 
+        if self.levels == 0:
```

Comment:
this is in Italo's PR too btw.

And it belongs to `def process` no?

Reply:
Fixed